### DISCUSSION TITLE
How to reset all settings?!

### DISCUSSION THREAD

**Discussion by mohas** [smplayer-dev/smplayer#542] [Q&A] [answered]
I don't know what I hit but now smplayer has gone into this theater view mode where seeker and toolbar is not visible, how can I reset all settings to factory settings?

**Comment by smplayer-dev** [comment, marked as answer]
I guess you switched to compact mode. Press Ctrl+C to turn it off.
To reset all settings you need to delete the configuration folder (or at least the file smplayer.ini). The location of this folder depends on the operating system, you can use the option "Open configuration folder" from the Help menu to find it.
